wormhole/send.go

```diff
@@ -323,6 +328,7 @@ func (c *Client) sendFileDirectory(ctx context.Context, offer *offerMsg, r io.Re
 
 		err = transport.listenRelay()
 		if err != nil {
+			fmt.Printf("error: %v\n", err)
```

Comment:
was it intended to be printed here. I guess later when the this function gets called and if the channel returns an error message, we may be printing it anyway?

Reply:
Ah this is probably a leftover artifact from the rebasing. I'll remove it